LICENSE

```diff
@@ -0,0 +1,11 @@
+Copyright (c) 2020-2022 Parmance Oy, Pagan LLC, Argonne
```

Comment:
Should probably be "..., Argonne National Laboratory"

Reply:
fixed